data/json/vehicleparts/combustion.json

```diff
@@ -339,7 +339,7 @@
     "difficulty": 7,
     "durability": 400,
     "epower": -10,
-    "power": 1006695,
+    "power": 1409373,
     "energy_consumption": 4026780,
```

Comment:
consumption should be 4x power.
there should probably be two different turbines here - the UH-60 and AH-64 both use the 1410 kW GE T700, while the Abrams MBT uses the 1120 kW Honeywell AGT1500C turbine.

Reply:
Yeah I edited the wrong one anyway, the apaches use the medium turbine which is already broadly correct.

Reply:
Ive edited it back.